emergence_lib/src/items.rs

```diff
@@ -173,6 +179,16 @@ impl ItemSlot {
     }
 }
 
+impl Display for ItemSlot {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "{} ({}/{})",
```

Comment:
Ditto here: will be much clearer with the variables inside the string.

Reply:
Here it's also not possible because of the `.` access.
Do you prefer me to move them into a variable and then put them in the string?

Reply:
That would be better, but I won't block on it.